spec/unit/blockscore/errors/api_connection_error_spec.rb

```diff
@@ -1,6 +1,6 @@
 module BlockScore
   RSpec.describe APIConnectionError do
-    let(:stubbed_route_pattern) { %r(.*api\.blockscore\.com/people/connection_refused) }
+    let(:stubbed_route_pattern) { %r{.*api\.blockscore\.com/people/connection_refused} }
```

Comment:
Style/PercentLiteralDelimiters: `%r`-literals should be delimited by `(` and `)`